### ISSUE TITLE
Fatal Error on Deploy as of today's version

### ISSUE THREAD

**Issue by cornelisonc** [DeltaSystems/delta-cli-tools#3]
Deploy appears to work correctly (ie, the files end up on the server), however it is terminating when it hits this error:

```
/delta/project delta deploy project
git-status-is-clean completed successfully.
git-branch-matches-environment completed successfully.
sync-project-theme completed successfully on staging.servers.deltasys.com.
  staging.servers.deltasys.com

Fatal error: Call to a member function getName() on a non-object in /Users/ceili/.composer/vendor/deltasystems/delta-cli/library/DeltaCli/Script/Step/Ssh.php on line 49
```

**Comment by griffbrad** [comment]
What is your delta-cli.php when running this?

**Comment by cornelisonc** [comment]
[OFA-Website on BitBucket](https://bitbucket.org/deltasystemsgroup/ofa-website/src/e42c859c3d92f5b9585ae730e1911251815b75df/delta-cli.php?at=develop&fileviewer=file-view-default)

**Comment by griffbrad** [comment]
Fixed in 1.17.1.
